.eslintrc.js

```diff
@@ -2,9 +2,11 @@ module.exports = {
   parser: '@typescript-eslint/parser', // Specifies the ESLint parser
   extends: [
     'plugin:@typescript-eslint/recommended', // Uses the recommended rules from the @typescript-eslint/eslint-plugin
-    'prettier/@typescript-eslint', // Uses eslint-config-prettier to disable ESLint rules from @typescript-eslint/eslint-plugin that would conflict with prettier
+    //'prettier/@typescript-eslint', // Uses eslint-config-prettier to disable ESLint rules from @typescript-eslint/eslint-plugin that would conflict with prettier
```

Comment:
Is there a reason why this is commented out? If we really don't need it anymore probably we should just delete this line?

Reply:
All prettier configs have been merged into just 'prettier' I will delete the commented line